stdlib/format.mli

```diff
@@ -508,16 +505,23 @@ val safe_set_geometry : max_indent:int -> margin:int -> unit
    [pp_set_geometry ppf ~max_indent ~margin] sets both the margin
    and maximum indentation limit for [ppf].
 
+   [margin] and [max_indent] should satisfy
+   [1 < max_indent < margin < 1,000,000,010].
```

Comment:
The exact numerical value is an implementation detail that is not part of the specification of the module.